modules/clightning-plugins/feeadjuster.nix

```diff
@@ -0,0 +1,61 @@
+{ config, lib, ... }:
+
+with lib;
+let cfg = config.services.clightning.plugins.feeadjuster; in
+let maybeFlag = enable: flag: if enable then flag + "\n" else ""; in
```

Comment:
There has to be a builtin for this, I just couldn't find it …

Reply:
Yes, there is - it's called `optionalString` see https://github.com/NixOS/nixpkgs/blob/772e39b23727a5e69bf021fe1bde079b965580e8/lib/strings.nix#L199

Reply:
Thx!